modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -253,8 +254,26 @@ private void _validate(
 		}
 
 		if (Objects.equals(
+				objectActionTriggerKey,
```

Comment:
Could you sort the conditions inside the IF?
Objects.equals(objectActionExecutorKey,...) before Objects.equals(objectActionTriggerKey,...)